scripts.js

```diff
@@ -0,0 +1,38 @@
+var prefix = "https://cors-anywhere.herokuapp.com/";
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+
+function getQuote() {
+    $.getJSON(prefix + quoteUrl, createTweet);
+    $.ajaxSetup({ cache: false });
+
+}
+
+function createTweet(input) {
+    var data = input[0];
```

Comment:
always check data that you receive - you need to be sure that input is an array

Reply:
how to check that data?